src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/repository.py

```diff
@@ -228,15 +229,129 @@ def acr_repository_show_manifests(cmd,
         result_index='manifests')
 
 
+def acr_repository_untag(cmd,
+                         registry_name,
+                         image,
+                         resource_group_name=None,
+                         username=None,
+                         password=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd.cli_ctx, registry_name, resource_group_name, UNTAG_NOT_SUPPORTED)
+
+    repository, tag, _ = _parse_image_name(image)
+
+    login_server, username, password = get_access_credentials(
+        cli_ctx=cmd.cli_ctx,
+        registry_name=registry_name,
+        resource_group_name=resource_group_name,
+        username=username,
+        password=password,
+        repository=repository,
+        permission='*')
+
+    return _delete_data_from_registry(
+        login_server=login_server,
+        path='/v2/_acr/{}/tags/{}'.format(repository, tag),
+        username=username,
+        password=password)
+
+
 def acr_repository_delete(cmd,
                           registry_name,
-                          repository,
+                          repository=None,
+                          image=None,
                           tag=None,
                           manifest=None,
                           resource_group_name=None,
                           username=None,
                           password=None,
                           yes=False):
+    if (not repository and not image) or (repository and image):
+        raise CLIError("Please specify either an image name with --image or a repository name with --repository.")
```

Comment:
Consider this, which is more consistent with the rest of the CLI:
```Python
if bool(repository) == bool(image):
  raise CLIError('usage error: --image IMAGE | --repository REPOSITORY')
```

The first line is a more concise way of expression XOR. Favor a 'usage' style message in your error.

Reply:
Thanks, fixed.